recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -512,10 +548,29 @@ CONFIG_DEFAULT_DEPMOD_FILE="modules.dep"
 #
 # CONFIG_BLOCKDEV is not set
 # CONFIG_FATATTR is not set
+# CONFIG_FSTRIM is not set
+# CONFIG_MDEV is not set
+# CONFIG_FEATURE_MDEV_CONF is not set
+# CONFIG_FEATURE_MDEV_RENAME is not set
+# CONFIG_FEATURE_MDEV_RENAME_REGEXP is not set
+# CONFIG_FEATURE_MDEV_EXEC is not set
+# CONFIG_FEATURE_MDEV_LOAD_FIRMWARE is not set
+CONFIG_MOUNT=y
```

Comment:
This was disabled by default before, and a USE flag exist to enable it. This change thus both changes the default and makes the flag not have any effect...